PluginCode/src/main/java/smr/cs/ualberta/libcomp/action/ReplacementAction.java

```diff
@@ -285,7 +292,11 @@ public void detectMaven(@NotNull final Editor editor, @NotNull final PsiFile psi
         String lineText;
         String selectedTerm;
 
-        TextAttributes attributes = EditorColorsManager.getInstance().getGlobalScheme().getAttributes(DebuggerColors.BREAKPOINT_ATTRIBUTES);
+        TextAttributes attributes = EditorColorsManager.getInstance().getGlobalScheme().getAttributes(EditorColors.WRITE_SEARCH_RESULT_ATTRIBUTES);
+        if (UIUtil.isUnderDarcula()) {
+            attributes.setBackgroundColor(JBColor.YELLOW);
```

Comment:
Let's stick to same color since IntelliJ doesn't change the color between themes (it already just changes the color slightly)